src/index.ts

```diff
@@ -451,7 +451,7 @@ class ExtensibilitySdk {
    * @returns {Promise<string | null>}
    * @memberof ExtensibilitySdk
    */
-  public authenticate = async (): Promise<string | null> => {
+  public authenticate = async (redirectUri?: string): Promise<string | null> => {
```

Comment:
There is one more optional parameter to append to popup URL that I believe will be useful especially in case of multiple redirectUris: [state parameter](https://medium.com/keycloak/the-importance-of-the-state-parameter-in-oauth-5419c94bef4c)

I therefore suggest adding it as optional here as well